api/app/src/routes/user.ts

```diff
@@ -151,6 +153,10 @@ router.delete(
       //   // await Constants.PROVIDER_OAUTH1_MAP[
       //   //   providerOAuth1.data
       //   // ].deregister(connectedUser);
+    } else if (req.query.provider === PROVIDER_APPLE) {
+      const apple = new Apple()
+      await apple.revokeProviderAccess(connectedUser);
+      return res.sendStatus(200);
     } else {
```

Comment:
I could add an provider "noauthschema" similar to garmin. I didnt know if that would be extra overhead just to add a revoke function but let me know.

Reply:
I think this is okay for now, we can see about doing something like this once we have multiple providers in the same boat